samples/WebFormsSample/Helpers.cs

```diff
@@ -1,8 +1,29 @@
 // MIT License.
 
+using System.Data.Entity;
+
 namespace WebFormsSample;
 
+//
 public class Helpers
 {
     public static string Message = "Hello from a helper";
 }
+
+public class SampleContext : DbContext
```

Comment:
It would be better to have this in the code behind or in a separate page so there is no chance for it to potentially be loaded. Having this helpers class will cause ef6 to be at least referenced by the main app

Reply:
I added in the codebehind. Didn't get the second statement :)